frontend/main.py

```diff
@@ -160,19 +160,21 @@ def get_queues():
 def resolve_correction(queue_name, correction_id):
     data = request.json
     if "action" not in data or (
-        data["action"] == "approve" and "corrected_val" not in data
+        data["action"] == "approve" and "corrected_val" not in data and "corrected_grade" not in data
```

Comment:
do you mean corrected_grade not needed here, because api will send the all the value for hadithText and grade in this attribute `corrected_val`?
can you help me to understand about the requirement 🙏🏽

Reply:
corrected_val can be used to keep the value of whatever attribute that has been corrected and we need another field to capture the name of the corrected attribute i.e. we'll have only two attributes attr and corrected_val.

You can take a look at the backend table to get an idea of what it is supposed to be like
https://github.com/sunnah-com/corrections/blob/master/frontend/scripts/init-dynamodb.sh

Reply:
ok jazakAllah for the explanation.
I have another question. after implement this idea, can we update both of attribute `grade` and `hadithText` in the same request?

Reply:
No we don't want to update both in the same request.

Reply:
hi @hasankhan, PR updated, could you help check again? jazakAllah